src/warc2zim/items.py

```diff
@@ -44,6 +44,10 @@ def __init__(
         else:
             self.content = self.record.content_stream().read()
 
+        should_rewrite = getattr(record, "method", "GET") != "POST"
```

Comment:
Should we not rewrite POST requests or only rewrite GET? I don't know how many PATCH/PUT/DELETE requests we might have (probably rare) but I don't see why they should be more rewriten than POST requests

Reply:
I agree. In fact, I think the condition is not right at all.
We should not rewrite content designated to be use by js (as ajax call response will already be rewritten by wombat).

However it seems their is not standard way to detect a request come from ajax. Maybe we have to work with the scrapper side (you) to mark records as coming from ajax (or simply as "don't rewrite").

But I propose to move this in another issue as we will probably don't fix this easily.

Reply:
We had this exact discussion on our call with Ilya. I remember there's no way to know it from the WARC but we envisioned a similar solution: have browsertrix add this information to it during crawl. The information is in the browser (even in the dev tools) but I don't know if it's exposed in their API and how easy it would be to implement.
Once you've opened the ticket here, we'll open a ticket upstream (browsertrix-crawler).

Reply:
Issue created : https://github.com/openzim/warc2zim/issues/140